src/main/java/com/itech/kinopoisk/entity/Film.java

```diff
@@ -55,6 +56,10 @@
     @NotBlank
```

Comment:
there should be NO underscores in the variable name. Only camelCase. (posterUrl, posterUrlPreview)